integreat_cms/cms/views/utils/search_content_ajax.py

```diff
@@ -105,6 +106,34 @@ def search_content_ajax(  # noqa: PLR0915, PLR0912, C901
     results: list[dict[str, Any]] = []
 
     user = request.user
+
+    if "contact" in object_types:
+        object_types.remove("contact")
+        if not user.has_perm("cms.view_contact"):
+            raise PermissionDenied
+        results.extend(
+            {
+                "title": contact.name,
+                "url": None,
+                "type": "contact",
+            }
+            for contact in Contact.objects.filter(
+                name__icontains=query, location__region=region, archived=archived_flag
+            ).distinct()
+        )
+        if TYPE_CHECKING:
+            assert language_slug
+        results.extend(
+            {
+                "title": poi_translation.title,
+                "url": None,
+                "type": "contact",
+            }
+            for poi_translation in POITranslation.search(
```

Comment:
Why POITranslation? What is the relation to contacts? Further down we do a similar query if pois are in object_types, but with more filters. It seems to me we should at least apply the same filters, and do a permission check for pois here as well.